htmresearch/frameworks/location/path_integration_union_narrowing.py

```diff
@@ -262,24 +264,36 @@ def learnObject(self, objectDescription):
     self.reset()
     self.column.activateRandomLocation()
 
-    for iFeature, feature in enumerate(objectDescription["features"]):
-      self._move(feature)
+    if randomLocation or useNoise:
+      numIters = 5
```

Comment:
This would be cleaner as a passed in argument that defaults to 1 but no need to change for this PR